content/teams/Electronics CreeD/index.md

```diff
@@ -0,0 +1,132 @@
+---
+type: "team"                                                        
+title: "Electronics Creed"
+name: "Electronics Creed"
+github-link: "https://github.com/Saikumarksk"
+description: "We are young developers who accept difficult challenges"
+Team_Tech_Stack: "Python, Django, HTML, CSS, Java Script"
+Application_Type: "Web Application"
+No_Of_Member: "5"
+---
+
+Hey ! We are Team DemoTeam. We are here to win. and we also love open source.
+
+---
+
+## Project's Overview
+
+_**Theme-Name**_ : AtmaNirbhar Bharat
+
+_**Application-Type**_ :   Web Application
+
+_**Idea-Description**_ :   The Web Application is about 
+
+_**Tech-Stack-Used**_ :   Python, Django, HTML, Java Script, CSS
+
+_**GitHub-Link**_ :   https://github.com/Saikumarksk/Code-Camp 
+
+_**Hosted-Link**_ :   https://github.com/Saikumarksk
```

Comment:
Hosted link is not needed now. Leave is empty or put N/A there.
Hosted link can not be your  github link it is to be a website URL or your real application live link